lib/resolver.js

```diff
@@ -165,7 +202,7 @@ SchemaResolver.prototype.resolveRef = function (ref) {
     return dest;
 };
 
-SchemaResolver.prototype.resolve = function (schema) {
+SchemaResolver.prototype.resolve = function (schema, missing$Ref) {
```

Comment:
I think it's better to put this `missing$ref` as an option in the constructor, because, after the first call to `resolve(schema, true)` it leaves artifacts (the `missing` array) that are persisted regardless of the value of `missing$Ref` on successive calls.